.ci/pipeline/ci.yml

```diff
@@ -60,12 +60,21 @@ jobs:
     vmImage: 'ubuntu-20.04'
   steps:
   - script: |
+      # Add bazel repository
+      sudo apt install curl gnupg
+      curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
+      sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
+      echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | \
+        sudo tee /etc/apt/sources.list.d/bazel.list
+
       sudo apt-get update
       sudo apt-get install gcc-multilib \
                            g++-multilib \
                            binutils \
                            openjdk-11-jdk \
-                           bazel-3.4.1
+                           bazel-4.0.0
```

Comment:
Might be good to get this regularly updated - for example via https://docs.renovatebot.com/modules/manager/regex/
This shouldn't block integraton